common/persistence/sql/storage/sqldb/interfaces.go

```diff
@@ -596,7 +597,11 @@ type (
 
 		InsertIntoCurrentExecutions(row *CurrentExecutionsRow) (sql.Result, error)
 		UpdateCurrentExecutions(row *CurrentExecutionsRow) (sql.Result, error)
+		// SelectFromCurrentExecutions returns one or more rows from current_executions table
+		// Required params - {shardID, domainID, workflowID}
 		SelectFromCurrentExecutions(filter *CurrentExecutionsFilter) (*CurrentExecutionsRow, error)
+		// DeleteFromCurrentExecutions performa a conditinal delete of a row in current_executions table
+		// Required params - {shardID, domainID, workflowID, runID}
```

Comment:
From the doc it is not clear what happens when condition doesn't hold. Does it return an error or doesn't delete and returns OK.

Reply:
you are right - let me update the doc - if the condition doesn't hold - it simply returns ok